src/test/java/com/thealgorithms/sorts/MergeSortTest.java

```diff
@@ -0,0 +1,91 @@
+package com.thealgorithms.sorts;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class MergeSortTest {
+
+    private static MergeSort mergeSort;
+    Integer[] array;
+    Integer[] expected;
+    Integer[] sorted;
+
+    @BeforeAll
+    public static void setup() {
+        mergeSort = new MergeSort();
+    }
+
```

Comment:
There's no need in these fields and test setup because the data is not shared between tests and there's no real setup to do

Reply:
Ok. WIll change that.